docs/tutorials/02a_training_a_quantum_model_on_a_real_dataset.ipynb

```diff
@@ -0,0 +1,891 @@
+      "text/plain": [
+       "<Figure size 1290.6x264.88 with 1 Axes>"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit.circuit.library import RealAmplitudes\n",
+    "\n",
+    "ansatz = RealAmplitudes(num_qubits=num_features, reps=3)\n",
+    "ansatz.decompose().draw(output=\"mpl\", fold=20)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "comic-bumper",
+   "metadata": {},
+   "source": [
+    "This circuit has 16 parameters names as `θ[0], ..., θ[15]`. These are our trainable weights of the classifier.\n",
+    "\n",
+    "Then, we choose an optimization algorithm we'll use in the training process. This is pretty similar to what you may find in classical deep learning frameworks. To make the training process faster we choose a gradient free optimizer. You may explore other optimizers available in Qiskit."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "intimate-doubt",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit.algorithms.optimizers import COBYLA\n",
+    "\n",
+    "optimizer = COBYLA(maxiter=100)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "existing-syracuse",
+   "metadata": {},
+   "source": [
+    "We also fix initial values for weights. This is the point the optimization process will start from."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "sixth-serial",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "\n",
+    "initial_point = np.asarray([0.5] * ansatz.num_parameters)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "integral-compound",
+   "metadata": {},
+   "source": [
+    "On the next step we define where our classifier will be trained. It can be a simulator or a real quantum computer. Here, we use a simulator. We fix the seeds for reproducibility purposes and as quantum computers are noisy and probabilistic by their nature we set shots to 1024 to sample results from many runs or shots of the same circuit."
```

Comment:
Idea was to show on a simulator that is close enough to real devices, so `qasm` simulator. By noisy I mean that they are NISQ devices. In general, I'm fine with this sentence, let me know how I can improve it.

Reply:
It's currently this right

>On the next step we define where our classifier will be trained. It can be a simulator or a real quantum computer. Here, we use a simulator. We fix the seeds for reproducibility purposes and as quantum computers are noisy and probabilistic by their nature we set shots to 1024 to sample results from many runs or shots of the same circuit.

I re-worded it a little around what I was thinking - maybe too much explanation now perhaps since it I kept the noise aspect which for the simulation chosen is kinda immaterial. But maybe it gives a bigger picture for real devices so is fine

> On the next step we define where our classifier will be trained. It can be a simulator or a real quantum computer. Here, we use a simulator, and being a simulation we can fix the random generator seeds for reproducibility purposes. Quantum computers are noisy and probabilistic by their nature. While the Aer qasm simulator can be configured to include noise in the simulation, here we do not but the outcome is still sampled i.e. probabilistic and we set shots to 1024 to sample results from many runs or shots of the same circuit.